core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -328,6 +328,8 @@
   USER_LOCAL_WRITER_PACKET_SIZE_BYTES(Name.USER_LOCAL_WRITER_PACKET_SIZE_BYTES, "64KB"),
   USER_NETWORK_NETTY_CHANNEL(Name.USER_NETWORK_NETTY_CHANNEL, null),
   USER_NETWORK_NETTY_TIMEOUT_MS(Name.USER_NETWORK_NETTY_TIMEOUT_MS, 30000),
+  USER_NETWORK_NETTY_WRITER_CLOSE_TIMEOUT_MS(Name.USER_NETWORK_NETTY_WRITER_CLOSE_TIMEOUT_MS,
+      30000),
```

Comment:
should we set this to a higher value (e.g. 5 minutes) since for object stores it could require a large file upload?